docs/vsc_ugent.md

```diff
@@ -0,0 +1,37 @@
+# nf-core/configs: University of Ghent High Performance Computing Infrastructure (VSC)
+
+> **NB:** You will need an [account](https://www.ugent.be/hpc/en/access/faq/access) to use the HPC cluster to run the pipeline.
+
+First you should go to the cluster you want to run the pipeline on. You can check what clusters have the most free space on this [link](https://shieldon.ugent.be:8083/pbsmon-web-users/). Use the following commands to easily switch between clusters:
+
+```shell
+module purge
+module swap cluster/<CLUSTER>
+```
+
+Before running the pipeline you will need to create a PBS script to submit as a job.
+
+```bash
+#!/bin/bash
+
+module load Nextflow
+
+nextflow run <pipeline> -profile vsc_ugent,<CLUSTER> <Add your other parameters>
+```
+
+I also highly recommend specifying a location of a Singularity cache directory, by specifying the location with the `$SINGULARITY_CACHEDIR` bash environment variable in your `.bash_profile` or `.bashrc` or by adding it to your PBS script. If this cache directory is not specified,
```

Comment:
> If this cache directory is not specified,

seems to kind of just trail off here - is it bad?? 😅

Reply:
Forgot to remove that part :p